cmd/frontend/internal/httpapi/webhookhandlers/handle_repo_authz_event.go

```diff
@@ -0,0 +1,55 @@
+package webhookhandlers
+
+import (
+	"context"
+	"fmt"
+	gh "github.com/google/go-github/v28/github"
+	"github.com/inconshreveable/log15"
+	"github.com/sourcegraph/sourcegraph/cmd/frontend/backend"
```

Comment:
It is cleaner to place stdlib, third-party libs and our packages in different groups.

```suggestion
	"fmt"

	gh "github.com/google/go-github/v28/github"
	"github.com/inconshreveable/log15"

	"github.com/sourcegraph/sourcegraph/cmd/frontend/backend"
```